benches/benches/invert.rs

```diff
@@ -0,0 +1,63 @@
+use ark_ff::Field;
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use lambdaworks_math::traits::ByteConversion;
+use rand::RngCore;
+
+const BENCHMARK_NAME: &str = "invert";
+
+pub fn criterion_benchmark(c: &mut Criterion) {
+    // arkworks-ff
+    {
+        use ark_std::{test_rng, UniformRand};
+        use ark_test_curves::starknet_fp::Fq as F;
+
+        let mut rng = test_rng();
+
+        let mut v = Vec::new();
+        for _i in 0..10000 {
+            let a = F::rand(&mut rng);
+            v.push(a);
+        }
+
+        c.bench_function(
+            &format!(
+                "{} | ark-ff - branch: faster-benchmarks-and-starknet-field",
+                BENCHMARK_NAME
+            ),
+            |b| {
+                b.iter(|| {
+                    for mut elem in v.iter_mut() {
+                        black_box(black_box(&mut elem).inverse_in_place());
+                    }
+                });
```

Comment:
Let's do:
```suggestion
                b.iter_batched(
                    || v.clone(),
                    |mut v| {
                        for mut elem in v.iter_mut() {
                            black_box(black_box(&mut elem).inverse_in_place());
                        }
                    },
                    BatchSize::LargeInput,
                );
```

Reply:
Otherwise you'll work on different inputs each iteration, while in the lambdaworks benchmark it uses always the same.

Reply:
Done